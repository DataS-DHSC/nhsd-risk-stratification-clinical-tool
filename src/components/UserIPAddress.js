import React from 'react';
import { ROUTE_IP_ENDPOINT } from '../constants/routes';
import { useFetch } from '../utils/hooks';

const UserIPAddress = () => {
  const [siteData, loading] = useFetch({
    url: ROUTE_IP_ENDPOINT,
    bodyType: 'text',
  });
  return <>{loading ? '' : siteData}</>;
};

export default UserIPAddress;
