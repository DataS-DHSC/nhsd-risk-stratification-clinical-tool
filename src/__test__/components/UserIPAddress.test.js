import React from 'react';
import { render } from '@testing-library/react';
import UserIPAddress from '../../components/UserIPAddress';

const setup = () => render(<UserIPAddress />);

describe('UserIPAddress', () => {
  it('should match snapshot ', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
