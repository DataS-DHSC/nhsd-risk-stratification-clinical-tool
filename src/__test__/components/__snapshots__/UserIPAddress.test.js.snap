// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserIPAddress should match snapshot  1`] = `<DocumentFragment />`;
